src/styles/watch/watch.scss

```diff
@@ -0,0 +1,39 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  height: $clock-size;
+  width: $clock-size;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.